src/main/webapp/resources/js/edit-booking.js

```diff
@@ -275,6 +280,12 @@ function refreshTable(bookingsState) {
             cell.innerHTML = i + 1;
         });
     }).draw();
+    $('.timepicker').timepicker({
```

Comment:
callback for draw() to make sure that DOM is ready before timepicker initialization